lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js

```diff
@@ -83,20 +84,37 @@ qx.Class.define("callbackery.ui.plugin.Action", {
                     break;
             }
         },this);
+        this.addListener('appear', () => {
+            let buttonMap = this._buttonUrlActionMap;
+            let config    = callbackery.data.Config.getInstance();
+            Object.entries(buttonMap).forEach(data => {
```

Comment:
if we were dealing with an array, this could would be a bit smpler

Reply:
Done.